.changelog/unreleased/improvements/997-mempool-spec.md

```diff
@@ -0,0 +1 @@
+- Add mempool specification in English and Quint ([#997](https://github.com/cometbft/cometbft/pull/997))
```

Comment:
I'd recommend adding spec changes to a dedicated "specification" section instead of "improvements", since spec updates don't influence the code at present and most readers of the changelog will be looking for changes to the software itself.

Reply:
Alright, I have removed the changelog entry. (BTW, there is still the one of the lightclient.)